index.html

```diff
@@ -204,8 +204,64 @@ <h5 class="modal__headline modal-title" id="ModalLabel">Design.</h5>
     </div>
 </main>
 
-    <footer>
-        
+    <footer class="footer" id="contact">
+      <div class="container">
+        <div class="row">
+          <div class="line-vertical col-12 text-center"></div> <!--pionowa kreska-->
+          <div class="col-12">
+            <h1 class="distinguish text-center mb-4">Kontakt</h1>
+            <p class="text-center mx-auto text-white  mb-5">Lorem ipsum dolor sit amet.</p>
+          </div>
+
+          <form action="#" class="form form-row text-white text-uppercase w-100">
+              <div class="form-group col-12 col-sm-6">
+                <label class="contact-label" for="FormControlInput">your name:</label>
```

Comment:
zle stworzona labelka, atrybut for musi wskazywac input, np.
`<label class="contact-label" for="name">your name:</label>
                <input type="text" id="name "class="form-item form-control">`